crates/ide/src/syntax_highlighting/highlight.rs

```diff
@@ -362,6 +375,10 @@ fn highlight_method_call(
     if func.is_unsafe(sema.db) || sema.is_unsafe_method_call(&method_call) {
         h |= HlMod::Unsafe;
     }
+    if let Some(_t) = func.as_assoc_item(sema.db)?.containing_trait(sema.db) {
```

Comment:
```suggestion
    if let Some(_) = func.as_assoc_item(sema.db).and_then(|it| it.containing_trait(sema.db)) {
```
We don't wanna cancel highlighting the method call just because the function isn't associated